samples/drivers/led_lp3943/README.rst

```diff
@@ -0,0 +1,46 @@
+.. _lp3943:
+
+LP3943: 16-Channel RGB, White-LED Driver
+########################################
+
+Overview
+********
+
+This sample display a continuous pattern that turns on 16 LEDs
+connected to LP3943 at one second interval one by one until it
+reaches the end and turns off LEDs in reverse order.
+
+Requirements
+************
+
+This sample uses the `LP3943 LED driver`_ controlled using the I2C interface.
```

Comment:
This app makes use of the onboard LED's. As mentioned in the overview, each LED will be turned ON at one second interval till it reaches the end. Then it will turn OFF in reverse order.